gofigure.go

```diff
@@ -242,32 +250,32 @@ func (gfg *gofiguration) initSources() error {
 
 func (gfg *gofiguration) registerFields() error {
 	for _, gfi := range gfg.fields {
-		for _, o := range gfg.order {
-			kn := gfi.field
-			if k, ok := gfi.keys[o]; ok {
-				kn = k
-			}
+		kn := gfi.field
 
-			gfg.printf("Registering '%s' for source '%s' with key '%s'", gfi.field, o, kn)
-			var err error
-			switch gfi.goField.Type.Kind() {
-			case reflect.Struct:
-				gfg.printf("Registering as struct type")
-				// TODO do shit
-				sGfg, err := parseStruct(gfi.goValue)
-				if err != nil {
-					return err
+		var err error
+		switch gfi.goField.Type.Kind() {
+		case reflect.Struct:
+			gfg.printf("Registering as struct type")
+			// TODO do shit
```

Comment:
Do we want to include this possibly remove it on our end to remove a certain word?